subiquity/models/filesystem.py

```diff
@@ -1711,6 +1756,12 @@ def _all(self, **kw):
     def all_mounts(self):
         return self._all(type='mount')
 
+    def all_mountlikes(self):
```

Comment:
I almost wonder if you should change all_mounts to do this. It would break the server tui in a more obvious way than the slightly subtle way leaving it alone does :-)